File: 백준/Bronze/4153. 직각삼각형/직각삼각형.js
var fs = require('fs');
var inputs = fs.readFileSync('/dev/stdin').toString().split('\n');
 
for(let line of inputs) {
  let values = line.split(' ').map(el => parseInt(el));
 
  if(values[0] === 0) break;
  // 정렬을 하는 이유는 제일 큰 숫자가 빗변이기 때문에.
  values.sort((a, b) => {
    return a - b;
  });
 
  // a^2+b^2=c^2이 성립하여야 직각이다.
  if((values[0] * values[0]) + (values[1] * values[1]) === (values[2] * values[2])) {
    console.log("right")
  } else {
    console.log("wrong")
  }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while(True):
  numbers = list(map(int, input().split(" ")))
  numbers.sort()
  a,b,c = numbers
  
  if(a == 0):
    break
  if(c**2 == a **2 + b ** 2) :
    print("right")
  else:
    print("wrong")
  